docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.7"
+
+services:
+  front:
+    image: caddy:2.2.1-alpine
+    environment:
+      URL:
+    ports:
+      - "80:80"
```

Comment:
should we add a domain name here and bind the port 443 as well so caddy can handle https traffic?

Reply:
there should be a separate version of the docker-compose file without caddy
as most users probably have a webserver/reverse proxy already setup

Reply:
Hi, it would be a good idea. But the base minimum configuration can be simplified for the general case of users. There will be a PR following this PR that allows user to get optional docker-compose configuration